website/src/components/navigation/sidebar/FrameworkSelect.tsx

```diff
@@ -42,7 +42,7 @@ export const FrameworkSelect = (props: FrameworkSelect) => {
   return (
     <Select
       defaultValue={defaultValue}
-      positioning={{ gutter: 2, placement: 'bottom-start', sameWidth: true }}
+      positioning={{ gutter: 2, sameWidth: true }}
```

Comment:
I think it is not related with page's scroll!

this placement means just select options placement 

```tsx
 /**
     * The initial placement of the floating element
     */
    placement?: Placement;
```

Reply:
<img width="260" alt="image" src="https://user-images.githubusercontent.com/69495129/232836431-ae7c181f-13b2-44fb-92d7-7342188bb25a.png">

when
```tsx
positioning={{ gutter: 2, placement: 'top-start', sameWidth: true }}
```

Reply:
Hi just wanted to provide evidance that removing the `placement` stops this from happening.
By default the placement is set to bottom.


https://user-images.githubusercontent.com/112592168/232896094-b7a96185-908d-4f59-b97a-d0603db8c790.mp4

Reply:
Holy..
Thanks for giving awesome video
Really It solved well..

I don’t know why they are related..(maybe portal?..)

Reply:
It does seem related to the Portal
On line 40 of `FrameworkSelect.tsx` there is `const SelectWrapper = props.noPortal ? Fragment : Portal`.
If you replace the SelectWrapper with a Fragment the issue doesn't happen. So it seems that the placement in the Select has a side effect on the Portal.